src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -250,6 +253,7 @@
     <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj" />
     <ProjectReference Include="..\..\System.Security.Principal\src\System.Security.Principal.csproj" />
     <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj" />
+    <ProjectReference Include="..\..\System.Buffers\src\System.Buffers.csproj" />
```

Comment:
Do you actually need to add this reference? Span is in `System.Runtime` for netcoreapp - you do not need to reference `System.Buffers` to get it.

Reply:
Ah, good catch, thanks! I needed in an earlier commit, because I was using `ArrayPool` directly, but that's been removed now.